docs/team/contributors-binder.yaml

```
@@ -4,6 +4,12 @@
   affiliation: UC Berkeley
   contributions: doc
   team: red
+  
+- name: Sarah Gibson
+  handle: "@sgibson91"
+  affiliation: The Turing Institute
```

```suggestion
  affiliation: The Alan Turing Institute
```
Just so our comms team don't get cross with me :)